app/(gcforms)/[locale]/(user authentication)/auth/mfa/actions.ts

```diff
@@ -119,7 +119,8 @@ export const verify = async (
     });
   } catch (err) {
     // Failed login attempt
-    if (err instanceof AuthError && err.name === "CredentialsSignin") {
+    logMessage.debug(err);
+    if ((err as AuthError).name === "CredentialsSignin") {
```

Comment:
AuthError is the base class of all Errors from Next-Auth which is also inherits the Error Class.
When cast to AuthError the `err.name` would be undefined if an unhandled error was thrown.
So `undefined === "CredentialSignin" is false which would then skip the block and return the general internal error.
If it would make it clearer I can also rewrite the typing to remove the cast if it would be easier to read?

Reply:
make sense! lgtm